dist/codicon.css

```diff
@@ -565,3 +565,4 @@
 .codicon-map-filled:before { content: "\ec06" }
 .codicon-circle-small:before { content: "\ec07" }
 .codicon-bell-slash:before { content: "\ec08" }
+.codicon-comment-unresolved:before { content: "\f101" }
```

Comment:
I think you forgot to add this to the mapping file? otherwise the unicode character will shift when adding new ones

Reply:
Whoops. Fixed this and did the same for `bell-slash-dot`